cmd/hyperkube/hyperkube.go

```diff
@@ -123,8 +123,8 @@ func (hk *HyperKube) Run(args []string) error {
 	command := args[0]
 	baseCommand := path.Base(command)
 	serverName := baseCommand
+	args = args[1:]
 	if serverName == hk.Name {
-		args = args[1:]
```

Comment:
This looks to perhaps break things when you softlink `kubectl` to the `hyperkube` binary.  Have you tested that?